release/ray_release/tests/test_test.py

```diff
@@ -204,5 +204,289 @@ def test_update_from_s3(mock_client) -> None:
     assert test["github_issue_number"] == "1234"
 
 
+@pytest.mark.parametrize(
+    "gen_s3_return, prefixes, expected_tests",
+    [
+        (
+            [
+                [
+                    Test(
+                        {
+                            "name": "linux://t1_s3",
+                            "team": "core",
+                            "state": TestState.FLAKY,
+                        }
+                    ),
```

Comment:
maybe create a helper function to create `Test`? so that this list looks a bit more compact / nicer, making it easier to read?